packages/shared-components/src/components/onto-repository-selector/onto-repository-selector.tsx

```diff
@@ -73,14 +82,18 @@ export class OntoRepositorySelector {
       return;
     }
 
-    this.items = this.attachTooltipsToItems(this.items);
+    this.dropdownItems = this.attachTooltipsToItems(this.items);
```

Comment:
So if I pass a new empty array, nothing will change and old items will be shown?

Reply:
You should clear `this.dropdownItems` in the above conditional before exiting

Reply:
+1